odoo/addons/base/models/ir_model.py

```diff
@@ -2201,6 +2200,9 @@ def delete(records):
         # remove models
         delete(self.env['ir.model'].browse(unique(model_ids)))
 
+        # announce undeletable ids
+        _logger.info(f"ir.model.data with the following IDs could not be deleted ({undeletable_ids})")
```

Comment:
Loggers should use `%s`-formatted strings
```suggestion
        # log undeletable ids
        _logger.info("ir.model.data could not be deleted (%s)", undeletable_ids)
```

Reply:
done, but can you explain why?